src/compiler/objective_c_plugin.cc

```diff
@@ -127,6 +127,10 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       }
     }
 
+    // Write out a file header.
+    ::grpc::string file_header = "// Code generated by gRPC proto compiler.  DO NOT EDIT!\n"
+        "// source: " + file->name() + "\n";
```

Comment:
Let's add an additional "\n" at the end for pretty

Reply:
Done!